### PR TITLE
Markdown

### PR DESCRIPTION
**Pull request by magsout**
I added new Component `Markdown` to handle _element_ from Github without ClassName. 

r? @miketaylr

### CODE REVIEW

**Comment by magsout** [comment]
Oops, neeed to rebase

**Comment by magsout** [comment]
@miketaylr I need you. I'm not an expert of git. When I try to fixed conflict after rebased I'm lost...

**Comment by miketaylr** [comment]
@magsout np! let me try to fix it up and push.

**Comment by miketaylr** [comment]
OK, should be good to review now. :+1:

**Comment by miketaylr** [comment]
Hm, weird error:

```
Running "cssnext:dist" (cssnext) task

Warning: /home/travis/build/webcompat/webcompat.com/webcompat/static/css/development/main.css:54:1: Failed to find 'components/Markdown' from /home/travis/build/webcompat/webcompat.com
```

`cssnext:dist` task works for me locally, maybe I need to kick the Travis cache.

File: webcompat/static/css/development/main.css [hunk 1]
```diff
@@ -51,3 +51,4 @@
 @import "components/label";
 @import "components/upload-form";
 @import "components/contributors";
+@import "components/Markdown";
```
Comment (miketaylr) [inline webcompat/static/css/development/main.css]: Maybe Travis doesn't like `Markdown` vs `markdown` here? (I guess Linux cares about case sensitivity and OSX doesn't? :confused:)

**Comment by magsout** [comment]
Ah yes, I usually write Markdown instead of markdown on my other project. But in this case `@import component/markdown` is good

**Comment by miketaylr** [comment]
LGTM. :shipit:

**Comment by magsout** [comment]
miketaylr  thanks, but curious, how did you do ?

**Comment by miketaylr** [comment]
Sure!

So to rebase against master from a branch (not a fork) I usually do something like this:

```
git fetch
git pull --rebase origin master
```

And then when there's a conflict I do `git status` to see which files were "both modified" (or whatever the message is).

I open those in my editor and search for `<<<<` to jump to the spot where the conflict is. This is the tricky part -- you have to decide which parts should stay and which parts go. This can be dead simple, or it can be confusing if patches change things around a lot.

https://help.github.com/articles/resolving-a-merge-conflict-from-the-command-line/#creating-an-edit-collision should help here a little. But it can still be a pain in the butt to get right.

Once you think you've got it right, you have to `git add file-that-you-just-edited` to let git know that you're done with the conflict resolution. And after that, `git rebase --continue`. 

You might get another conflict from the next patch it applies in the rebase. So rinse and repeat until it tells you you're done.

**Comment by magsout** [comment]
>Once you think you've got it right, you have to git add file-that-you-just-edited to let git know that you're done with the conflict resolution. And after that, git rebase --continue. 

OK, what I missed when I tried to rebase... Thanks for the explanation, :+1:

**Pull request merged**